weblate/accounts/templatetags/authnames.py

```diff
@@ -84,8 +84,11 @@ def get_auth_params(auth: str):
     return params
 
 
+auth_name_default_separator = format_html("<br />")
+
+
 @register.simple_tag
-def auth_name(auth: str, separator: str = "<br />"):
+def auth_name(auth: str, separator: str = auth_name_default_separator):
```

Comment:
I think it should be `mark_safe` then instead, since it's a dynamic string. Just so it's easier to see that you have to be careful to not pass a dangerous (user-provided) value to `auth_name`.